### ISSUE TITLE
[iOS] object cannot be nil (key: origURI)

### ISSUE THREAD

**Issue by iamjbecker** [marcshilling/react-native-image-picker#180]
Hey guys!

Just implemented this on iOS.

**The issue:** I only get this issue when I select `Take Photo`. And only upon tapping `Use this photo`.

    *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** setObjectForKey: object cannot be nil (key: origURI)'
    *** First throw call stack:
          (0x1838f2e38 0x182f57f80 0x1837d8554 0x10017f594 0x188d99854 0x188eca1f0 0x188d9ca4c 0x188b6cfd8 0x188b6cdc4 0x188b6c8c0 0x188b6c420 0x188a77b10 0x188a77998 0x1863e5a20 0x100345a3c 0x10034b4e4 0x1838a8dd8 0x1838a6c40 0x1837d0d10 0x1850b8088 0x188aa5f70 0x100051080 0x18336e8b8)
    libc++abi.dylib: terminating with uncaught exception of type NSException

**The code:** The following is triggered by a `<TouchableHighlight />`.

    ImagePickerManager.showImagePicker(options, (response) => {
      console.log('Response = ', response);

      if (response.didCancel) {
        console.log('User cancelled image picker');
      }
      else if (response.error) {
        console.log('ImagePickerManager Error: ', response.error);
      }

      else {
        var source = {};
        source.data = {uri: 'data:image/jpeg;base64,' + response.data, isStatic: true};
        source.uri = {uri: response.uri.replace('file://', ''), isStatic: true};

        this._takePicture({
          data: source.data.uri,
          uri: source.uri.uri
        });
      }
    });

I know my code works since it works on the Choose from Library.

**Additional Info:** When I select 'Take Photo', I get this warning.
```
Snapshotting a view that has not been rendered results in an empty snapshot. Ensure your view has been rendered at least once before snapshotting or snapshot after screen updates.
```

**Comment by li1** [comment]
Hi
Same issue here; I guess something broke with this commit: https://github.com/marcshilling/react-native-image-picker/commit/26ea4d579abf9244ef9623fdd964b40635fbc8ef

@iamjbecker As temporary workaround you can use version 0.18.3 instead of 0.18.4 (e.g. using npm i react-native-image-picker@0.18.13 --save ).

**Comment by marcshilling** [comment]
Fixed in [`v0.18.15`](https://github.com/marcshilling/react-native-image-picker/releases/tag/v0.18.15)

**Comment by Deevent** [comment]
I'm facing this issue after upgrading to react-native 0.44.3
